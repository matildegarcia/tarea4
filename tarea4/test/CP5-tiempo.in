colaPrioridadTiempo 110000 1 2

Fin
