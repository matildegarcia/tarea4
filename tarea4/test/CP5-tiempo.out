1>El jugador prioritario es: 
Jugador 1: Juan, Uruguay, 21 aÃ±os
Bien.
2>Fin.
